packages/ozone/tests/get-lists.test.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  SeedClient,
+  TestNetwork,
+  TestOzone,
+  basicSeed,
+  ModeratorClient,
+  RecordRef,
+} from '@atproto/dev-env'
+import AtpAgent from '@atproto/api'
+import { TAKEDOWN_LABEL } from '../src/mod-service'
+
+describe('admin get lists', () => {
+  let network: TestNetwork
+  let ozone: TestOzone
+  let agent: AtpAgent
+  let appviewAgent: AtpAgent
+  let sc: SeedClient
+  let modClient: ModeratorClient
+  let alicesList: RecordRef
+
+  beforeAll(async () => {
+    network = await TestNetwork.create({
+      dbPostgresSchema: 'ozone_admin_get_lists',
+    })
+    ozone = network.ozone
+    agent = ozone.getClient()
+    appviewAgent = network.bsky.getClient()
+    sc = network.getSeedClient()
+    modClient = ozone.getModClient()
+    await basicSeed(sc)
+    alicesList = await sc.createList(sc.dids.alice, "Alice's List", 'mod')
+    AtpAgent.configure({ appLabelers: [ozone.ctx.cfg.service.did] })
```

Comment:
Just a note that this config change may apply to other test suites using `AtpAgent`.  Perhaps we could revert it in `afterAll()`.